### PR TITLE
added possibility to set styles to SVG layer

### PR DESCRIPTION
**Pull request by ShvedDmutro**
# Checklist

- [ ] The code has been run through pretty `yarn run pretty`
- [ ] The tests pass on CircleCI
- [ ] You have referenced the issue(s) or other PR(s) this fixes/relates-to
- [ ] The PR Template has been filled out (see below)
- [ ] Had a beer/coffee because you are awesome

## What?
Possibility of setting styles to SVG layer

## Why?
Dynamically add the image to have possibility drag and drop nodes over the image on special areas of an image can be  different diagrams like a map

## Feel good image:

(Add your own one below :])

LOL

### CODE REVIEW

File: packages/react-canvas-core/src/entities/layer/TransformLayerWidget.tsx [hunk 1]
```diff
@@ -59,8 +60,10 @@ export class TransformLayerWidget extends React.Component<TransformLayerWidgetPr
 	}
 
 	render() {
+		const svgLayerStyles = this.props.svgLayerStyles || {};
+
 		if (this.props.layer.getOptions().isSvg) {
-			return <S.SvgLayer style={this.getTransformStyle()}>{this.props.children}</S.SvgLayer>;
+			return <S.SvgLayer style={{ ...svgLayerStyles, ...this.getTransformStyle()}}>{this.props.children}</S.SvgLayer>;
```
Comment (dylanvorster) [inline packages/react-canvas-core/src/entities/layer/TransformLayerWidget.tsx]: would actually prefer this to be done via `className={this.props.className}`

**Comment by dylanvorster** [comment]
While I agree in theory, I kind of think the stying should happen behind all the layers sine the goal of the transform layer is simply to position elements

File: packages/react-canvas-core/src/entities/layer/TransformLayerWidget.tsx (hunk 1 above)
Comment (ShvedDmutro) [reply packages/react-canvas-core/src/entities/layer/TransformLayerWidget.tsx]: the point is to have possibility dynamically add background and sizes depends on image size

Comment (dylanvorster) [reply packages/react-canvas-core/src/entities/layer/TransformLayerWidget.tsx]: Yeah so, ideally what we actually want to do here is let the developer render a new simple layer behind the nodes (and) links (which can be achieved using the api's in react-canvas)

Comment (dylanvorster) [reply packages/react-canvas-core/src/entities/layer/TransformLayerWidget.tsx]: (which can be dynamic)

Comment (ShvedDmutro) [reply packages/react-canvas-core/src/entities/layer/TransformLayerWidget.tsx]: Is there a sample? Also, I set an image as a background to keep all zooming, dragging functionality works

**Pull request closed without merging**